manifest.webapp

```diff
@@ -22,9 +22,15 @@
   },
   "permissions": {
     "apps": {
-      "description": "Required by the cozy-bar to display the icons of the apps",
+      "description":
+        "Required by the cozy-bar to display the icons of the apps",
       "type": "io.cozy.apps",
       "verbs": ["GET"]
+    },
+    "contacts": {
+      "description": "Required to to share files with your contacts",
```

Comment:
There's a double `to`, and I think `Required to manage your contacts` is probably a better description.